package.json

```diff
@@ -31,6 +31,7 @@
   "homepage": "http://material-ui.com/",
   "dependencies": {
     "inline-style-prefixer": "^0.6.6",
+    "lodash.flowright": "^3.2.1",
```

Comment:
I think 3.2.1 is still the latest version of the `lodash.flowright` package in npm. Not sure exactly what that means though!